examples/hello-world/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "@zeit/ncc": "file:../../"
```

Comment:
I'm installing ncc from the root of the repo - thoughts?

Reply:
Sounds good!

Reply:
One issue I'm noticing here is that if I run `yarn install` in the `examples/hello-world` directory, it can take ~2 minutes to install `@zeit/ncc` from two directories above.

If I change the version from `file:../../` to `latest` and install from npm, it downloads much faster.

[I notice Next.js' examples use `latest`](https://github.com/zeit/next.js/blob/cd1d3640a9abfa3df3cadf712fe095aa5326a089/examples/using-preact/package.json#L11), and perhaps that's a pattern we could adopt here. Or maybe there's a way to improve the speed of installing a file dependency with yarn using [a command flag](https://yarnpkg.com/lang/en/docs/cli/install/). If you have suggestions about next steps with this, let me know. 😄

Reply:
Let's do `latest` @macklinu. If we install all the dev deps from the directory above, it'll take a very long time. We have all those deps for integration testing purposes only

Reply:
@macklinu ping

Reply:
I addressed this in de48c58, but I don't think GitHub automatically resolved this thread. 😅